db/migrations/01_create_chitter_table.sql

```diff
@@ -1 +1,3 @@
-CREATE TABLE peeps(id SERIAL PRIMARY KEY, message VARCHAR(60));
+--OG version: CREATE TABLE peeps(id SERIAL PRIMARY KEY, message VARCHAR(60));
+CREATE TABLE peeps(id SERIAL PRIMARY KEY, message VARCHAR(60), post_date DATE NOT NULL default CURRENT_DATE);
```

Comment:
Recreating your table from scratch to add a new column is not great as it requires the table to be deleted first (you couldn't do that if this were a live app with real users!). The better approach is to use an ALTER TABLE statement to add a column to an existing table.

Good use of the DATE data type combined with CURRENT_DATE though!